README.md

```diff
@@ -20,11 +20,12 @@ Speed up your websites with a Compute@Edge environment that demonstrates serving
 
 ## Usage
 
-When initializing your project, the Fastly CLI will prompt you to enter a `Backend`. You can enter your bucket host here, or just enter `mock-s3.edgecompute.app` on port `443` if you want to experiment with our mock backend.
+When deploying your project, the Fastly CLI (`v1.1.0`) will prompt you to enter a `Backend`. You can enter your bucket host here, or just enter `mock-s3.edgecompute.app` on port `443` if you want to experiment with our mock backend.
```

Comment:
@triblondon let me know if specifying the version of the CLI in this way is OK.

Reply:
@triblondon In a separate repo you suggested we get into the habit of updating CLI instructions after each new release, so I presume the same principle applies to the starter kits.